wandb/integration/openai/fine_tune.py

```diff
@@ -0,0 +1,418 @@
+import datetime
+import io
+import json
+import re
+import time
+from typing import Any, Dict, Optional, Tuple
+
+from pkg_resources import parse_version
+
+import wandb
+from wandb import util
+from wandb.data_types import Table
+from wandb.sdk.lib import telemetry
+from wandb.sdk.wandb_run import Run
+
+openai = util.get_module(
```

Comment:
if you already doing the check in the __init__.py file it is not need here...

Reply:
you are right. fixing.